src/content/options.js

```diff
@@ -0,0 +1,13 @@
+export default function processOptions({ json, prettier: prettierOptions }) {
+  let options = {};
+
+  if (json.enable) {
```

Comment:
I'd prefer to check the presence of `json.config` directly so we don't need an extra variable and this state will have a single source of truth.

Reply:
I don’t think this would be a great user experience, because it means you have to clear out the JSON config manually even if you uncheck the option to use the JSON config.

Reply:
To be clear, I mean we would also remove the checkbox option.

Reply:
Can you explain why you think that would be better, from a user experience standpoint?

Reply:
I changed my mind, we don't need to store this config twice. We can use `options`/`prettierOptions`/whatever to store the parsed config for both the form and JSON editor. If you'd like we can also have a synced option to toggle whether the form or JSON editor is used, otherwise it would default to one of them and keep the state locally in React.